Core/include/Acts/Propagator/EigenStepper.hpp

```diff
@@ -100,20 +101,11 @@ class EigenStepper {
       }
     }
 
-    /// Global particle position
-    Vector3D pos = Vector3D(0., 0., 0.);
-
-    /// Momentum direction (normalized)
-    Vector3D dir = Vector3D(1., 0., 0.);
-
-    /// Momentum
-    double p = 0.;
-
-    /// The charge
-    double q = 1.;
+    /// Internal free vector parameters
+    FreeVector pars = FreeVector::Zero();
 
-    /// Propagated time
-    double t = 0.;
+    /// The charge as the free vector can be 1/p or q/p
+    int q = 1;
```

Comment:
I thought we decided to use `double` here to be consistent?

Reply:
Oh yes, will revert this.